<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手-客户端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>493</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="widgetVL" stretch="5,1,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextEdit" name="showTextEdit"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,3,2,1,3,2,3,2">
       <item row="0" column="1">
        <widget class="QLabel" name="ipServerLabel">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>服务端IP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="portServerLabel">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>服务端端口</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="HS2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="7" rowspan="2">
        <layout class="QVBoxLayout" name="connectVL">
         <item>
          <widget class="QPushButton" name="connectPushButton">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="interruptPushButton">
           <property name="text">
            <string>断开</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="6">
        <spacer name="HS4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="ipClientLabel">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>客户端IP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="ipServerLineEdit">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="HS6">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="portClientLineEdit">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>8889</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="portServerLineEdit">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>8888</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="HS5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <spacer name="HS1">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="portClientLabel">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>客户端端口</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="HS3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="8">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="8">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="ipClientComboBox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sendHL" stretch="3,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QTextEdit" name="sendTextEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPushButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
